src/main/java/com/coveros/selenified/utilities/Reporter.java

```diff
@@ -108,7 +110,7 @@
      */
     @SuppressWarnings("squid:S00107")
     public Reporter(String directory, String test, Capabilities capabilities, String url, String suite, String group,
-                    String author, String version, String objectives) throws InvalidBrowserException {
+                    String author, String version, String objectives) throws InvalidBrowserException, InvalidProxyException, InvalidHubException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.coveros.selenified.exceptions.InvalidHubException', as it cannot be thrown from constructor's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)